packages-web/badge-button/src/BadgeButton.xml

```diff
@@ -1,89 +1,48 @@
-                <enumerationValue key="modal">Blocking popup</enumerationValue>
-            </enumerationValues>
-        </property>
-        <property key="microflow" type="microflow" required="false">
-            <caption>Microflow</caption>
-            <category>Events</category>
-            <description>The microflow that will be executed on click</description>
-            <returnType type="Void" />
-        </property>
-        <property key="nanoflow" type="nanoflow" required="false">
-            <caption>Nanoflow</caption>
-            <category>Events</category>
-            <description>The nanoflow that will be executed on click</description>
-            <returnType type="Void" />
-        </property>
+        <propertyGroup caption="General">
+            <propertyGroup caption="Button">
+                <property key="label" type="textTemplate" required="false">
+                    <caption>Caption</caption>
+                    <description>The caption of the button</description>
+                    <translations>
+                        <translation lang="en_US">Button</translation>
+                        <translation lang="nl_NL">Knop</translation>
+                    </translations>
+                </property>
+                <property key="bootstrapStyle" type="enumeration" defaultValue="default">
+                    <caption>Button style</caption>
+                    <description>The appearance of the badge button</description>
+                    <enumerationValues>
+                        <enumerationValue key="default">Default</enumerationValue>
+                        <enumerationValue key="primary">Primary</enumerationValue>
+                        <enumerationValue key="success">Success</enumerationValue>
+                        <enumerationValue key="info">Info</enumerationValue>
+                        <enumerationValue key="warning">Warning</enumerationValue>
+                        <enumerationValue key="danger">Danger</enumerationValue>
+                    </enumerationValues>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Badge">
+                <property key="value" type="textTemplate" required="false">
+                    <caption>Badge value attribute</caption>
+                    <description>The attribute that contains the badge button value</description>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Events">
+                <property key="onClickEvent" type="action" required="false">
+                    <caption>On click</caption>
+                    <description />
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Common">
+                <systemProperty key="Name" />
+                <systemProperty key="Visibility" />
+            </propertyGroup>
+        </propertyGroup>
     </properties>
 </widget>
```

Comment:
Properties should align better with, core button widgets. Suggestion:

Please note added the `Inverse` as well
```xml
<?xml version="1.0" encoding="utf-8"?>
<widget id="com.mendix.widget.web.badgebutton.BadgeButton" pluginWidget="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../node_modules/mendix/custom_widget.xsd">
    <name>Badge button v2</name>
    <description/>
    <icon>iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAMAAACdt4HsAAACplBMVEUAAAD///+AgIBVVVVV//+AgICAv/9mZmZmzP9VVVVgv/9VVVVmZmZduf9Vv/9Osf9iYmJJtv9bW1taWlpVVVVVuP9eXl5ZWVlJtv9VVVVZWVlKtf9SZmZOsf9Ms/9VVVVbW1tPuf9YWFhYWFhVVVVLtPhJtvhVVVVMs/hNsvlZWVlIsflZWVlXV1dKtfpYWFhXV1dLtPpVVVVYWFhXV1dVVVVXV1dVVVVWVlZLtPtYWFhWVlZVVVVMs/dVVVVLtPhJsvhIs/hWVlZVVVVKsvVVVVVKsvlWVlZIsflWVlZJs/lXV1dXV1dWVlZKsvpXV1dOgaRKsvdXV1dWVlZVVVVWVlZJsvhVVVVWVlZVVVVWVlZXV1dJsfhWVlZWVlZWVlZWVlZIsPlVVVVKsflWVlZJsPlVVVVWVlZVVVVVVVVWVlZWVlZVVVVVVVVWVlZWVlZWVlZVVVVWVlZVVVVWVlZWVlZVVVVWVlZQcYdWVlZWVlZWVlZSdItJsvlWVlZJsfdWVlZIsfdWVlZWVlZJsfhVVVVVVVVJsPhIsfhVVVVWVlZIsfhJsPhWVlZVVVVVVVVIsPhJsPhVVVVIsfhVVVVWVlZVVVVVVVVWVlZVVVVJsfdVVVVIsfhWVlZLnNRTXmZWVlZVVVVJsPhWVlZIsfhWVlZIsfhWVlZIsfhWVlZVVVVWVlZVVVVWVlZVVVVJsfdVVVVVW15WVlZIsPdJsfdJsfdWVlZJsfdJsfdWVlZIsPdVVVVVVVVUZnJWVlZVVVVVVVVWVlZVW15VVVVWVlZJsfhVVVVKpOFVVVVIsfhJo+BKpeRVW19WVlZVVVVVVVVIsPhWVlZVVVVVVVVJsPhWVlZIsfhVVVVIsPdKodxKpOFLm9FMlchQepZTaHZUXWNVVVVVWl19DoylAAAA2HRSTlMAAQIDAwQEBQUGCAkKCwwNDQ4OERISExQVFRcYGRobGxwdHSAhIiMkJSsrLi4vMDEyMzM0NTY4OTs9PT4/QEJHSUpKS0xRU1NVVldYW1xdXl9gYWJjZWZmaGxucHNzdHZ3eHh5eXt7fX6BgoOEh4iJjI2PkJGSk5SVlZqbnJ6eoaSmpqerq62vsLGytLW1tre4ubq+v8HCw8TFy8zNzc/Q0NLT09TW19nb3N7f4OLj5OTk5ebn6Ojq6+vs7O3u7u/w8fLz9PX19vb39/f39/j5+vr7/P39/v7kR+qdAAACxElEQVR42u2V+VdNURTHv0+eSkSEzFPmDCVkJjJEpooMmYeHpJQpZA6ROfNMRApF9NBAMkaGp0No+0/s895qPe+t5XmnfuCHPmvdde7Z6+7PvfuedfZBNdX8XzQMT842kF08OhvhCWuCc0mFdyscYEEkqZLkaPF+Umf97/Xnyq/SeWlhD5pO03KIqNTHHJot831hP80yOGO7eZ7MUx1UGMEZBeb/mM1TL6igfcEpbVCBXH8tlLjAKT1RATFQ4zSn+FgL6sTGxkSEuPOdV1zc2hqAX2xs9IzekHjptsTPbc8RJvhPAneSZDYBYshAA4EwksRrgPkfifE3RRJsCKaEysc06ZRIa6Qgb+xOohD4GujEuIlRNWTE37+XDUH4Mipuij5UOIQyNfx4NpBCiYik9w0ctFopyA8K6mq7hPHASjrgXER+JsEGuogEOoZiosOmEhbaEESvJkM/XKfjS2/TKpPgJO1nyQ0cTTMKnurWTbIhGOBcSHO6k5F0DQscR3+gxZhOFICZLJBKvd6GICWbSgctp7zQ0CVEfcPIUEB0zQUuN4nScowCQ4YQNgT0cFsAUuRCOeRTFFf89nJUfQDt9hTRk9fnnDhCRsH5sm/Wgr+gaez6QOx24jsp8HguyhQFTA+92MVDaircrogv5SxQ3UxsaA64umK4eFXOm0l9O7cdDNTuXBO1Rr7hlNaVaiiTn4ms/tYNJdyOlhb4UgixDx1KeMhqcecrD/puKk3VKNiLCULy+KdZoNjWvwvJD8u2zkQqC5IcrY82JYE82tQPV7Ngoycqy1Qh8ca/Flxa0BHq1A0cWs8k+MzXqVmuUOSQEPdGGQUmx61WUKKlTPp0UJjZBCXcSlhgwVWosVlYcQZqeNy3EmyFImOEZRHzoMoOYcEwoGpF8DJWqQg9mKoUcQSVwOOuWbAIlaGRdwVdNKimmgp+AfwMb+k/OIPLAAAAAElFTkSuQmCC</icon>
    <properties>
        <propertyGroup caption="General">
            <propertyGroup caption="General">
                <property key="label" type="textTemplate" required="false">
                    <caption>Caption</caption>
                    <description/>
                    <translations>
                        <translation lang="en_US">Button</translation>
                        <translation lang="nl_NL">Knop</translation>
                    </translations>
                </property>
                <property key="bootstrapStyle" type="enumeration" defaultValue="default">
                    <caption>Button style</caption>
                    <description/>
                    <enumerationValues>
                        <enumerationValue key="default">Default</enumerationValue>
                        <enumerationValue key="inverse">Inverse</enumerationValue>
                        <enumerationValue key="primary">Primary</enumerationValue>
                        <enumerationValue key="info">Info</enumerationValue>
                        <enumerationValue key="success">Success</enumerationValue>
                        <enumerationValue key="warning">Warning</enumerationValue>
                        <enumerationValue key="danger">Danger</enumerationValue>
                    </enumerationValues>
                </property>
            </propertyGroup>
            <propertyGroup caption="Badge">
                <property key="value" type="textTemplate" required="false">
                    <caption>Value</caption>
                    <description/>
                </property>
            </propertyGroup>
            <propertyGroup caption="Visibility">
                <systemProperty key="Visibility" />
            </propertyGroup>
            <propertyGroup caption="Events">
                <property key="onClickEvent" type="action" required="false">
                    <caption>On click</caption>
                    <description />
                </property>
            </propertyGroup>
            <propertyGroup caption="Common">
                <systemProperty key="Name" />
                <systemProperty key="TabIndex"/>
            </propertyGroup>
        </propertyGroup>
    </properties>
</widget>
```

Reply:
Done

Reply:
: D did you test style inverse?